miniapps/nurbs/nurbs_ex1.cpp

```diff
@@ -121,11 +121,50 @@ class Diffusion2Integrator: public BilinearFormIntegrator
 
 };
 
+void RefineNURBSFromFile(std::string ref_file, Mesh *mesh)
```

Comment:
There are no new sample runs for this example file. Is the new code necessary only for some mesh that is not in the repo? If so, can you please provide a minimal example mesh file, and add a corresponding sample run? That will ensure the new features are tested.

Reply:
Hi @dylan-copeland,

Thank you for your review! I did this to be able to insert a knot during the testing stage to show that knot insertion works correctly and to create an asymmetric mesh. I have added a test which uses this function: `nurbs_ex1_two_squares_rot_knot_insert`. I will add a sample run to the file as well.

Kind regards,

Jacob